.github/workflows/main.yml

```diff
@@ -188,6 +189,13 @@ jobs:
         ## Unzip and remove the artifact
         unzip artifacts.zip
         rm artifacts.zip
+    - name: initialize vcpkg
+      shell: cmd
+      working-directory: ./compat/vcbuild
+      run: |
+        echo Fetching vcpkg in %cwd%vcpkg
+        git.exe clone https://github.com/Microsoft/vcpkg vcpkg
+        IF ERRORLEVEL 1 ( EXIT /B 1 )
```

Comment:
I think we could use [`actions/checkout`](https://github.com/actions/checkout/) here, simplifying the added code (and accelerating it by the implicit shallow clone).

@dennisameling what do you think?

Reply:
Done @dscho, the `initialize vcpkg` step now takes 12s instead of 6-7s, but I guess that's okay 😄

Reply:
I think that time is very variable, anyway.